configs/lanl/config.MatchBoth_orig

```diff
@@ -127,4 +127,4 @@ baseDirectory = /usr/projects/climate/SHARED_CLIMATE/ACMEv0_lowres/B1850C5_ne30_
 ## circulation (MOC)
 
 # Mask file for ocean basin regional computation
-regionMaskFiles = /turquoise/usr/projects/climate/SHARED_CLIMATE/mpas_analysis/region_masks/EC60to30wLIv1_SingleRegionAtlantic_new_transect_masks.nc
+regionMaskFiles = /turquoise/usr/projects/climate/SHARED_CLIMATE/mpas_analysis/region_masks/oEC60to30v3_Atlantic_region_and_southern_transect.nc
```

Comment:
@milenaveneziani, what do you think of renaming the mask files to have the same names (suffix `_SingleRegionAtlantic_new_transect_masks.nc`) as on NERSC?  I think this would be good so all the user has to set is the name of the grid and the region mask gets found automatically (or there's an error if it's not there).  Obviously, that doesn't happen in the current PR but it's something I have coming down the pipeline...

Reply:
In principle I would say yes, but right now we have a confusion of MOC transect files out there. So much that I am not sure which ones don't really work, and I don't want to delete them for fear of getting rid of the right ones..
We need to solve this once and for all, getting the right MOC files, including the right workflow.
I would like to get to the bottom of this issue right after these outstanding PR's are in.
What do you think?